File: ISA-Frontend/src/app/model/adminRegistrationDTO.ts
export class AdminRegistrationDTO {
    email = ''
    password = ''
    userType = 'ADMIN'
    name = ''
    surname = ''
    gender = ''
  
  
    public constructor(obj?: AdminRegistrationDTO) {
      if (obj) {
        this.email = obj.email
        this.password = obj.password
        this.name = obj.name
        this.surname = obj.surname
        this.gender = obj.gender
      }
    }
  }
  

File: ISA-Backend/BloodBankAPI/Materials/DTOs/AdminRegistrationDTO.cs


using System.ComponentModel.DataAnnotations;

namespace BloodBankAPI.Materials.DTOs
{
    public class AdminRegistrationDTO
    {
        [Required]
        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }
        [Required]
        public string Password { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Surname { get; set; }
        [Required]
        public string Gender { get; set; }

        public AdminRegistrationDTO()
        {

        }
    }
}
